src/zod.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  appendErrors,
+  FieldValues,
+  Resolver,
+  ResolverError,
+  ResolverSuccess,
+  transformToNestObject,
+} from 'react-hook-form';
+import { Schema, ZodError } from 'zod';
+import { ParseParams } from 'zod/lib/src/parser';
+import convertArrayToPathName from './utils/convertArrayToPathName';
+
+const parseErrorSchema = (
+  zodError: ZodError,
+  validateAllFieldCriteria: boolean,
+) => {
+  if (!Array.isArray(zodError.errors) || zodError.isEmpty) {
```

Comment:
```suggestion
  if (!zodError.isEmpty) {
```

I think we can trust zod typings to be true. errors is always an array

Reply:
you can! :)